kore/src/Kore/Repl/Data.hs

```diff
@@ -442,8 +445,9 @@ helpText =
     \<alias>                                  runs an existing alias\n\
     \load file                                loads the file as a repl script\n\
     \proof-status                             shows status for each claim\n\
-    \log [<severity>] \"[\"<entry>\"]\"           configures logging; <severity> can be debug ... error; [<entry>] is a list formed by types below;\n\
-    \    <type> <switch-timestamp>            <type> can be stderr or 'file filename'; <switch-timestamp> can be (enable|disable)-log-timestamps;\n\
+    \log [<severity>] [<format>]              configures logging; <severity> can be debug ... error; <format> can be standard or oneline;\n\
+    \    \"[\"<entry>\"]\"                    [<entry>] is a list formed by types below;\n\
+    \    <type> [<switch-timestamp>]          <type> can be stderr or 'file filename'; <switch-timestamp> can be (enable|disable)-log-timestamps;\n\
```

Comment:
When I type `help` in `kore-repl` the text looks like this
```
log [<severity>] [<format>]              configures logging; <severity> can be debug ... error; <format> can be standard or oneline;
    "["<entry>"]"                    [<entry>] is a list formed by types below;
    <type> [<switch-timestamp>]          <type> can be stderr or 'file filename'
```
Shouldn't `[<entry>] is a list formed by types below;` be aligned with the other lines?

Reply:
Yes, that happens because of the escaping backslashes in the string. I'll fix that after the standup.